tests/test_duplicate_mediafile.py

```diff
@@ -0,0 +1,63 @@
+import psycopg2
+import pytest
+import requests
+
+DUPLICATE_URL = "http://media:9006/internal/media/duplicate_mediafile/"
+GET_URL = "http://media:9006/system/media/get/"
+
+
+@pytest.fixture(autouse=True)
+def reset_db_2():
+    """Deletes all mediafiles except for id=2 and id=3 (example data)"""
+    conn = psycopg2.connect(
+        host="media-postgresql",
+        port=5432,
+        database="openslides",
+        user="openslides",
+        password="openslides",
+    )
+    with conn:
+        with conn.cursor() as cur:
+            cur.execute("DELETE FROM mediafile_data WHERE id NOT IN (2, 3)")
```

Comment:
Will this be often used in other places, too? Then extract this into an own function, too.

Reply:
Extracted into reset_db function.